test/it/ServicesISpec.scala

```diff
@@ -29,26 +31,54 @@ import scala.util.Try
 class ServicesISpec extends PlaySpec with GuiceOneAppPerSuite with TryValues {
   private val statementParser = new AccessibilityStatementParser
 
-  "the configuration files" should {
-    val sourceConfig   = app.injector.instanceOf[SourceConfig]
+  "parsing the configuration files" should {
+    val sourceConfig = app.injector.instanceOf[SourceConfig]
     val servicesFinder = app.injector.instanceOf[ServicesFinder]
 
     servicesFinder.findAll().foreach { (service: String) =>
       val source = sourceConfig.statementSource(service)
-
       val statementTry = Try(statementParser.parseFromSource(source).valueOr(throw _))
 
-      s"include a correctly formatted accessibility statement yaml file for $service" in {
+      s"enforce a correctly formatted accessibility statement yaml file for $service" in {
         statementTry must be a 'success
       }
 
-      s"not contain missing milestones for $service" in {
+      s"enforce statement not contain missing milestones for $service" in {
         val statement: AccessibilityStatement = statementTry.get
-
         val hasMilestones = statement.milestones.getOrElse(Seq.empty).nonEmpty
-
         hasMilestones || statement.isNonCompliant || statement.isFullyCompliant must be(true)
       }
+
+      s"enforce serviceDomain in the format of aaaa.bbbb.cccc for $service" in {
+        val domainRegex = "([a-z0-9-]*[\\.]*)*[a-z0-9]*"
+        val statement: AccessibilityStatement = statementTry.get
+        statement.serviceDomain.matches(domainRegex) must be(true)
+      }
+
+      s"enforce serviceUrl starting with / for $service" in {
+        val statement: AccessibilityStatement = statementTry.get
+        statement.serviceUrl.startsWith("/") must be(true)
+      }
+    }
+  }
+
+  "the file names in the directory" should {
```

Comment:
I wondered if we could remove a bit of duplication here by using the findAll() method in ServicesClasspathFinder and just check if the supplied filename is a member?